libs/perun/components/src/lib/vos-list/vos-list.component.html

```diff
@@ -15,12 +15,12 @@
       matSortDisableClear>
       <ng-container matColumnDef="checkbox">
         <th *matHeaderCellDef mat-header-cell class="align-checkbox"></th>
-        <td *matCellDef="let member" class="static-column-size align-checkbox" mat-cell>
+        <td *matCellDef="let vo" class="static-column-size align-checkbox" mat-cell>
           <mat-checkbox
-            (change)="$event ? selection.toggle(member) : null"
+            (change)="$event ? selection.toggle(vo) : null"
             (click)="$event.stopPropagation()"
-            [aria-label]="checkboxLabel(member)"
-            [checked]="selection.isSelected(member)"
+            [aria-label]="selection.isSelected(vo) | checkboxLabel | translate: {name: vo.name ?? vo.vo.name}"
```

Comment:
It's also a little strange that the name parameter is cut after a few letters. For example instead of `Select item Vo with a very very long name` there is a label `Select item Vo with a very ver` (it occurs not only on vos list - attributes have usually long names). IMHO it's not a big problem, but it is strange :D.

Reply:
I am not sure why is the text cut. As far as I know the aria-label and translation pipe is not restricted by lenght.